WebDevStudios/wds-acf-blocks
# v2.0.0 Update to use Portable blocks; also known as ACF Block Starter

No ticket. Uses the new ACF Block Starter framework/methodology. 

### DESCRIPTION ###
- Completely rebuilt the plugin
- Allows for blocks to be built in an Atomic-style methodology
- uses "portable" blocks, built using `wp-scripts`

### DOCUMENTATION ###
https://github.com/WebDevStudios/wds-acf-blocks.wiki.git
(pull the feature/wiki-updates)

### STEPS TO VERIFY ###
Follow the documentation and perform the following tasks:
* pull the plugin to a Local environment
* build some blocks, either manually or using the WP cli command
* style the newly built block, verify that the style is working as expected on the front & backend
* test with a theme that is not wd_s (ie: twenty twenty)
* test with wd_s
* verify that the plugin still works (tailwind specifically) without the theme having a tailwind preset
* test that the mobile styles have a basic preset
